datadog_checks_base/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Security***:
+
+* Update in-toto version ([#15667](https://github.com/DataDog/integrations-core/pull/15667))
```

Comment:
Small nit: we should update the description since this PR is also bumping securesystemslib.